libyul/backends/evm/EVMDialect.cpp

```diff
@@ -302,6 +317,9 @@ BuiltinFunctionForEVM const* EVMDialect::builtin(YulString _name) const
 
 bool EVMDialect::reservedIdentifier(YulString _name) const
 {
+	if (m_objectAccess)
+		if (regex_match(_name.str(), verbatimPattern()))
```

Comment:
Oh, actually shouldn't we also disallow `verbatim` as a precaution? And how about `verbatim*` or `verbatim_*` ?

Reply:
Ok, I'll change it to disallow `verbatim*`